.github/rt-dictionary.yaml

```diff
@@ -0,0 +1,4 @@
+# This file is a dictionary used by the [link-dependencies](https://github.com/newrelic/release-toolkit/tree/main/link-dependencies) action.
+
+dictionary:
+  common-library: "https://github.com/newrelic/helm-charts/releases/tag/common-library-{{.To}}"
```

Comment:
Here we will need to add all the known dependencies that are not produced in absolute format, do we already know any looking at the existing dependabot/renovate commits?

Reply:
TBH i don't know how this work. Could you point me to the docs?

Reply:
Here is the WIP Readme.md https://github.com/newrelic/release-toolkit/tree/add_readme#linked-changelog

Reply:
not sure if this make sense for the go dependencies. we can inspect the bots pr for that